xml/security_pcidss.xml

```diff
@@ -281,75 +281,71 @@
      </varlistentry>
      <varlistentry xml:id="vle.pcidss.firewall">
       <term>
-       1.2.1.b Examine firewall and router configurations to verify that
-       inbound and outbound traffic is limited to that which is necessary for
-       the cardholder data environment.
+       1.2.1.b Verify that inbound and outbound traffic is limited to
+       that which is necessary for the cardholder data environment.
       </term>
-      <!-- FIXME: SLE 15 will remove SuSEFirewall2 in favor of firewalld. -
-      sknorr, 2017-12-08 -->
       <listitem>
        <para>
-        Inbound traffic filtering rules can be defined from the &yast;
-        firewall module. Systems with multiple interfaces can be configured
-        to make the SSH daemon only reachable on the administration interface
-        and not on the general network card.
-        Furthermore, it is possible to define the source addresses that a
-        service allows traffic from.
+        Outbound traffic should only be allowed in specifically defined
+        cases. To allow specific traffic, manually add rules for
+        outbound traffic.
        </para>
        <para>
-        Usually all outbound system traffic is allowed by the
-        &susefirewall; script. Therefore outbound rules need to be defined
-        manually inside the &susefirewall; custom script. Activate the custom
-        script inside the general &susefirewall; configuration file
-        <filename>/etc/sysconfig/SuSEfirewall2</filename> by uncommenting
-        the <literal>FW_CUSTOMRULES</literal> line.
-       </para>
-       <para>
-        To add an outbound rule, add an appropriate
-        <command>iptables</command> command line inside the
-        <literal>fw_custom_after_chain_creation()</literal> function. This
-        function hook is executed during the firewall setup and allows any
-        customized <systemitem>iptables</systemitem> rule.
+        Make the SSH daemon only reachable on the administration
+        interface and not on the general network card if possible.
+        Define the source addresses that a service allows traffic from.
        </para>
        <para>
         For example, to allow only outbound DNS requests over the
         interface <systemitem>eth0</systemitem> to server
-        <systemitem class="ipaddress">10.0.0.4</systemitem>, use:
+        <systemitem class="ipaddress">10.0.0.1</systemitem>, use:
        </para>
-       <screen>&prompt.sudo;iptables -A OUTPUT -d 10.0.0.4/32 -o eth0 -p udp -m udp --dport 53 -j ACCEPT</screen>
```

Comment:
The new firewalld rule does not specify an interface.

`firewall-cmd --permanent --direct --add-rule ipv4 filter OUTPUT 23 -o eth0 -d 10.0.0.1/32 -p udp -m udp --dport 53 -j ACCEPT`

This rule has only an affect on the outgoing traffic if we set the general OUTPUT policy to DROP.  See 1.2.1.c.

Reply:
The priority (in my example 23) must be between the connection tracking priority and the DROP rule priority.

Reply:
For the record: added an -o eth0 and updated the priority.